src/app/src/components/DownloadFacilitiesButton.jsx

```diff
@@ -59,56 +61,73 @@ function DownloadFacilitiesButton({
     };
 
     return (
-        <div>
-            <Button
-                variant="outlined"
-                styles={downloadFacilitiesStyles.listHeaderButtonStyles}
-                aria-owns={anchorEl ? 'download-menu' : undefined}
-                aria-haspopup="true"
-                onClick={handleClick}
-                style={{
-                    backgroundColor: '#FFCF3F',
-                    fontSize: '16px',
-                    fontWeight: 900,
-                    lineHeight: '20px',
-                }}
-            >
-                <div
+        <Tooltip
```

Comment:
I'm seeing the tooltip all the time, regardless of whether the download button is enabled or the large downloads flag is enabled. Can it be hidden except for when the button in disabled?

Reply:
@TaiWilkin that's what I did originally, but then I saw [this](https://github.com/open-apparel-registry/open-apparel-registry/issues/2081#issuecomment-1252689299). I agree, though, and I think that it definitely makes sense to hide when the flag is set to true. Should I only hide it when the flag is true, or whenever the button is disabled?

Reply:
Ah, I missed that. Good catch. I think it's okay to leave the tooltip regardless of whether the button is disabled, then. So only hide it when the flag is true.